src/precommit-entry.js

```diff
@@ -33,9 +33,12 @@ export function getCommitMsg(path) {
       fsp.readFile(path, {encoding: 'utf8'})
     ])
     .then(([projectKey, fileContents]) => {
+      commitMsg = fileContents;
        let issues = getIssueReference(fileContents, projectKey);
        return issueHandler.issueStrategizer(issues, jiraAPI);
-    });
+    })
+    .catch(error => Promise.reject(`Commit Message:\n${commitMsg}\n${error}`
```

Comment:
By doing this you are bypassing all the error formatting below. Would probably make sense to merge the two functions together now.